Branch2018/test1/Program.cs

```diff
@@ -0,0 +1,195 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace test1
+{
+    class Program
+    {
+        //public List<Venta> ArraynuevaVenta = new List<Venta>();
+        //public List<Cliente> ArrayclienteNuevo = new List<Cliente>();
+        static void Main(string[] args)
+        {
+            
+             int _OpcionMenu = 0;            
+            while(_OpcionMenu!=6){

```

Comment:
I sent you an email where says that must to be an space before to a keyword like while, for and others.

Reply:
ok, ready